cpp/bench/sg/fil.cu

```diff
@@ -36,6 +36,7 @@ struct Params {
   ML::fil::storage_type_t storage;
   ML::fil::algo_t algo;
   RF_params rf;
+  int predict_times;
```

Comment:
I would change this to predict_nreps or predict_repetitions. The word "time" is too overloaded in the benchmark context.

Reply:
yes, thank you!

Reply:
Of course, something like `predict_num_reps` or `predict_num_repetitions` is also possible.